inc/zoo/swar/SWAR.h

```diff
@@ -362,6 +365,39 @@ constantIsGreaterEqual_MSB_off(SWAR<NBits, T> subtrahend) noexcept {
     }
 }
 
+template<typename T, typename U, typename V>
+constexpr T median(T x, U y, V z) {
+  return (x | y) & (y | z) & (x | z);
+}
+
+template<int NBits, typename T>
+constexpr BooleanSWAR<NBits, T>
+greaterEqual(SWAR<NBits, T> left, SWAR<NBits, T> right) noexcept {
+    // Adapted from TAOCP V4 P152
+    // h is msbselector, x is right, l is lower/left.  Sets MSB to 1 in lanes
+    // in test variable t for when xi < yi for lane i . Invert for greaterEqual.
+    // t = h & ~<x~yz>
+    // z = (x|h) - (y&~h)
+    using S = swar::SWAR<NBits, T>;
+    const auto h = S::MostSignificantBit, x = left.value(), y = right.value();  // x=left, y= right is x < y
+    const auto z = (x|h) - (y&~h);
+    // bitwise ternary median! 
+    const auto t = h & ~median(x, ~y, z);
+    return ~BooleanSWAR<NBits, T>{static_cast<T>(t)};  // ~(x<y) === x >= y
+}
+
+// In the condition where only MSBs will be on, we can fast lookup with 1 multiply the index of the leftmost byte.
```

Comment:
It is an intentionally terrible name (Knuth uses lambda and rho), and the intended use is so I can throw a less than and then know which lane failed PSL check first... 😄 
I don't think I like calling it log, but let me contemplate it some more.

Reply:
I think the constants given map 1:1 a pattern like "100'100'100'100" (MSB only on in all lanes or nothing on in lane) to [0,2^(NumLanes)) but I don't yet understand it enough to test and use the generalization.
Also, throwing to a table lookup feels like it will only be useful for smaller SWAR ops.

For some reason I thought we already had SWAR logarithm somewhere?